### PR TITLE
olx.com.br tracker and geolocation domains

### PR DESCRIPTION
**Pull request by fabianski7**
- the domain lurker.olx.com.br is called by the listing.js script
- the geoip.olx.com.br domain is self-explanatory, it captures the most precise location of the user, city and state.

<details><summary>Screenshots</summary>

ksnip_tmp_rBxdqy

ksnip_20211110-211043

</details>

### CODE REVIEW

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -144,8 +144,11 @@ fategrandorder.info##.sidebar_acr.side_inline
 ||dojq4kt8ws9iq.cloudfront.net/js/analytics.js
 ||components2.gazzettaobjects.it/rcs_tealium/
 ||liquipedia.net/stats/
-||static.olx.com.br/olx/js/lurker.min.js
+||lurker.olx.com.br^
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:147]: wrong file => tracking_servers_firstparty.txt

File: SpywareFilter/sections/specific.txt [hunk 2]
```diff
@@ -144,8 +144,11 @@ fategrandorder.info##.sidebar_acr.side_inline
...
+||geoip.olx.com.br^
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:148]: not tracking. Look request content.

File: SpywareFilter/sections/specific.txt [hunk 3]
```diff
@@ -144,8 +144,11 @@ fategrandorder.info##.sidebar_acr.side_inline
...
+||pub.olx.com.br/listing.js
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:149]: What the purpose?

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/specific.txt (hunk 3 above)
Comment (fabianski7) [reply SpywareFilter/sections/specific.txt:156]: This is script carries advertisements and is also what initiates contact with lurker.olx.com.br
https://bin.disroot.org/?72fd632e1069e9ae#2KBqqNrT4V8eVbQbFc4uahW86eNkkF9HudT2qTid8Ty7

File: SpywareFilter/sections/specific.txt (hunk 2 above)
Comment (fabianski7) [reply SpywareFilter/sections/specific.txt:155]: geoip.olx.com.br/ip identifies the user's most accurate location and displays ads specific to that region. The second screenshot shows this (obviously I hid my location)

Comment (Alex-302) [reply SpywareFilter/sections/specific.txt:155]: It uses info about IP. They know it in any case. This is used not for tracking.

Comment (fabianski7) [reply SpywareFilter/sections/specific.txt:155]: Yes, of course. but blocking the ip will prevent the site from displaying targeted advertising. Especially when using the mobile app, as it's not possible to apply css filters inside it.

Comment (Alex-302) [reply SpywareFilter/sections/specific.txt:155]: You [probably] will see just non-relevant ads(created by users on this site).

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**